content/en/serverless/troubleshooting/installing_the_forwarder.md

```diff
@@ -1,354 +0,0 @@
----
```

Comment:
this is supposed to delete because we're single sourcing it from the repo - we don't want it to appear in this repo too or people will try to update it here